.editorconfig

```diff
@@ -20,7 +20,7 @@ charset = utf-8
 indent_size = 4
 # isort plugin configuration
 known_first_party = invenio_records_permissions
-multi_line_output = 2
+multi_line_output = 5
```

Comment:
Hmm... This comes from `cookiecutter-invenio-module`. If it should be changed, could you make a PR there? if its a local default I think we should avoid pushing them (maybe add to the .gitignore?)